homeassistant/components/zamg/sensor.py

```diff
@@ -176,6 +196,12 @@ class ZamgSensorEntityDescription(SensorEntityDescription, ZamgRequiredKeysMixin
         col_heading="Zeit",
         dtype=str,
     ),
+    ZamgSensorEntityDescription(
+        key="station_id",
+        name="Station id",
+        col_heading="Station",
+        dtype=str,
+    ),
```

Comment:
Perhaps this should be in a different PR as the title suggests we are just adding a config flow here.

Reply:
I agree, its also redundant information of this `station_id`.
the `station_id` is already in the attributes of every sensor